gitignore

```diff
@@ -1,4 +1,3 @@
-!tags/
```

Comment:
@geoffharcourt I was unable to declare `tags` and `!tags/` in a way that they both co-existed. The only way I was able to successfully `ag` code in those files was without both lines.

It looks like this line was moved _above_ the line it negates in https://github.com/thoughtbot/dotfiles/commit/013f9711affdebc592b9010e916681858922aab8, which is a commit you authored.

Do you remember what motivated the re-ordering?